### PR TITLE
[storage] table info index try 2

### PR DESCRIPTION
**Pull request by msmouse**
### Description

Brutal force parsing table info from the write set.

### Test Plan

updated api::test::test_get_table_item

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2191)

### CODE REVIEW

File: aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs [hunk 1]
```diff
@@ -1233,7 +1235,7 @@ pub fn stake_update_network_and_fullnode_addresses(
 }
 
 /// Withdraw from `account`'s inactive stake.
-pub fn stake_withdraw() -> TransactionPayload {
+pub fn stake_withdraw(withdraw_amount: u64) -> TransactionPayload {
```
Comment (CapCap) [inline aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs:1238]: are these changes required for the table?

**Review by CapCap — COMMENTED** [review]
it's beautiful

**Review by CapCap — APPROVED** [review]

File: aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs (hunk 1 above)
Comment (msmouse) [reply aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs:1238]: no, rebase issue (someone force pushed main? seeing it elsewhere as well.)

**Comment by nkysg** [comment]
In this https://github.com/aptos-labs/aptos-core/issues/1293, mentioned that  

New table handles being created are part of the table extension output (TableChangeSet) https://github.com/move-language/move/blob/43f1e151dea91ca8e8c7644557e55b8f456fbb65/language/extensions/move-table-extension/src/lib.rs#L181
```rust
#[derive(Default)]
pub struct TableChangeSet {
    pub new_tables: BTreeSet<TableHandle>,
    pub removed_tables: BTreeSet<TableHandle>,
    pub changes: BTreeMap<TableHandle, TableChange>,
}
```

but right now I find that  https://github.com/aptos-labs/aptos-core/blob/aptos-node-v1.5.3/third_party/move/extensions/move-table-extension/src/lib.rs#L78

TableChangeSet member new_tables add TableInfo
```rust
/// A table change set.
#[derive(Default)]
pub struct TableChangeSet {
    pub new_tables: BTreeMap<TableHandle, TableInfo>,
    pub removed_tables: BTreeSet<TableHandle>,
    pub changes: BTreeMap<TableHandle, TableChange>,
}
```
I think https://github.com/aptos-labs/aptos-core/blob/aptos-node-v1.5.3/aptos-move/aptos-vm/src/move_vm_ext/session.rs#L275 in function convert_change_set can get table info .
Is it possible to save table info directly.

 Please take a look.  @msmouse
